src/foapy/alphabet.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+
+"""
+Implementation of ordered set - alphabet of elements.
+Alphabet is list of all unique elements in particular sequence.
+Parametres
+—--------
+X: array
+Array to get unique values.
+
+Returns
+—-----
+result: array.
+
+Examples
+—------
+
+"""
+
+
+def alphabet(X) -> np.ndarray:
+    result = []
+
+    for i in X:
+        if i not in result:
+            result.append(i)
+
+    return result
+
+
+print("_______1_____")
+X = ["a", "c", "c", "e", "d", "a"]
+result = alphabet(X)
+print(result)
+
+
+print("_______2_____")
+X = [0, 1, 2, 3, 4]
+result = alphabet(X)
+print(result)
+
+
+print("_______3_____")
+X = []
+result = alphabet(X)
+print(result)
```

Comment:
Remove this test code. 
You can use tests for such purposes. 
Remember about [TDD](https://ru.wikipedia.org/wiki/%D0%A0%D0%B0%D0%B7%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D0%BA%D0%B0_%D1%87%D0%B5%D1%80%D0%B5%D0%B7_%D1%82%D0%B5%D1%81%D1%82%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5)